docs/docs/user/migration-guide/v2-v3.md

```diff
@@ -63,3 +63,21 @@ steps:
       command: deploy --execute
       # ...
 ```
+
+## [Allow overriding config files](https://github.com/bakdata/kpops/pull/391)
+
+Specifying the environment is no longer mandatory. If not defined, only the default files will be used.
```

Comment:
```suggestion
Specifying the environment is no longer mandatory. If not defined, only the global files will be used.
```

perhaps like this to avoid confusion with `defaults.yaml`?